common/changes/@typespec/compiler/feature-string-template_2023-11-14-00-00.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@typespec/compiler",
+      "comment": "**New language feature** **BREAKING** Added string template literal in typespec. Singel and multi-line strings can be interpolated with `${` and `}`. Example `\\`Doc for url ${url} is here: ${location}\\``",
```

Comment:
```suggestion
      "comment": "**New language feature** **BREAKING** Added string template literal in typespec. Single and multi-line strings can be interpolated with `${` and `}`. Example `\\`Doc for url ${url} is here: ${location}\\``",
```

Reply:
Would also be good to explain the breaking change somewhere probably.

Reply:
yeah that's where we need a better changelog management tool

Reply:
For now Ill just file our breakiung change issue with the text to add in the reelease notes